include/verilated_funcs.h

```diff
@@ -1586,46 +1586,67 @@ static inline WDataOutP VL_STREAML_WWI(int lbits, WDataOutP owp, WDataInP const
 }
 
 template <typename T>
-static inline void VL_ASSIGN_DYN_Q(VlQueue<T>& q, int elem_size, int lbits, QData from) {
-    const int size = (lbits + elem_size - 1) / elem_size;
-    q.renew(size);
-    const QData mask = VL_MASK_Q(elem_size);
-    for (int i = 0; i < size; ++i) q.at(i) = (T)((from >> (i * elem_size)) & mask);
+static inline IData VL_PACK_II(int obits, int lbits, const VlQueue<T>& q) {
+    IData ret = 0;
+    for (int i = 0; i < q.size(); ++i) ret |= q.at(i) << (i * lbits);
```

Comment:
```suggestion
    for (size_t i = 0; i < q.size(); ++i) ret |= q.at(i) << (i * lbits);
```

Several others below too - realize this was wrong originally but might as well clean up.